testsuite/MDAnalysisTests/analysis/test_contacts.py

```diff
@@ -394,63 +621,228 @@ def test_warn_deprecated_attr(self, universe, client_Contacts):
-                   0.95095368, 0.94187103, 0.95186194, 0.94277929, 0.94187103,
-                   0.9373297, 0.93642144, 0.93097184, 0.93914623, 0.93278837,
-                   0.93188011, 0.9373297, 0.93097184, 0.93188011, 0.92643052,
-                   0.92824705, 0.92915531, 0.92643052, 0.92461399, 0.92279746,
-                   0.92643052, 0.93278837, 0.93188011, 0.93369664, 0.9346049,
-                   0.9373297, 0.94096276, 0.9400545, 0.93642144, 0.9373297,
-                   0.9373297, 0.9400545, 0.93006358, 0.9400545, 0.93823797,
-                   0.93914623, 0.93278837, 0.93097184, 0.93097184, 0.92733878,
-                   0.92824705, 0.92279746, 0.92824705, 0.91825613, 0.92733878,
-                   0.92643052, 0.92733878, 0.93278837, 0.92733878, 0.92824705,
-                   0.93097184, 0.93278837, 0.93914623, 0.93097184, 0.9373297,
-                   0.92915531, 0.93188011, 0.93551317, 0.94096276, 0.93642144,
-                   0.93642144, 0.9346049, 0.93369664, 0.93369664, 0.93278837,
-                   0.93006358, 0.93278837, 0.93006358, 0.9346049, 0.92824705,
-                   0.93097184, 0.93006358, 0.93188011, 0.93278837, 0.93006358,
-                   0.92915531, 0.92824705, 0.92733878, 0.92643052, 0.93188011,
-                   0.93006358, 0.9346049, 0.93188011]
-    assert_allclose(q1q2.results.timeseries[:, 1], q1_expected, rtol=0, atol=1.5e-7)
-
-    q2_expected = [0.94649446, 0.94926199, 0.95295203, 0.95110701, 0.94833948,
-                   0.95479705, 0.94926199, 0.9501845, 0.94926199, 0.95387454,
-                   0.95202952, 0.95110701, 0.94649446, 0.94095941, 0.94649446,
-                   0.9400369, 0.94464945, 0.95202952, 0.94741697, 0.94649446,
-                   0.94188192, 0.94188192, 0.93911439, 0.94464945, 0.9400369,
-                   0.94095941, 0.94372694, 0.93726937, 0.93819188, 0.93357934,
-                   0.93726937, 0.93911439, 0.93911439, 0.93450185, 0.93357934,
-                   0.93265683, 0.93911439, 0.94372694, 0.93911439, 0.94649446,
-                   0.94833948, 0.95110701, 0.95110701, 0.95295203, 0.94926199,
-                   0.95110701, 0.94926199, 0.94741697, 0.95202952, 0.95202952,
-                   0.95202952, 0.94741697, 0.94741697, 0.94926199, 0.94280443,
-                   0.94741697, 0.94833948, 0.94833948, 0.9400369, 0.94649446,
-                   0.94741697, 0.94926199, 0.95295203, 0.94926199, 0.9501845,
-                   0.95664207, 0.95756458, 0.96309963, 0.95756458, 0.96217712,
-                   0.95756458, 0.96217712, 0.96586716, 0.96863469, 0.96494465,
-                   0.97232472, 0.97140221, 0.9695572, 0.97416974, 0.9695572,
-                   0.96217712, 0.96771218, 0.9704797, 0.96771218, 0.9695572,
-                   0.97140221, 0.97601476, 0.97693727, 0.98154982, 0.98431734,
-                   0.97601476, 0.9797048, 0.98154982, 0.98062731, 0.98431734,
-                   0.98616236, 0.9898524, 1.]
```

Comment:
Leave original?

These values smell a bit. Is this an expected result computed analytically or is it just the result of funning the analysis made into a reproducibility test?